wiki/Tournaments/SOFT/6/en.md

```diff
@@ -0,0 +1,189 @@
+---
+tags:
+  - SOFT6
+  - SOFT 6
+---
+
+# Springtime osu!mania Free-for-all Tournament 6
+
+![SOFT6 logo](img/logo.jpg)
+
+> We actually thought this was impossible...
+
+The **Springtime osu!mania Free-for-all Tournament 6** (***SOFT 6***) is a 1v1 country-independent osu!mania 4-key tournament.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-01-29/2022-02-11 (23:59 UTC) |
+| Qualifiers showcase | 2022-02-19 (16:00 UTC) |
+| Qualifiers | 2022-03-05/2022-03-06 |
+| Playoffs week 1 | 2022-03-12/2022-03-13 |
+| Playoffs week 2 | 2022-03-19/2022-03-20 |
+| Group stage week 1 | 2022-03-26/2022-03-27 |
+| Group stage week 2 | 2022-04-02/2022-04-03 |
+| Round of 32 | 2022-04-09/2022-04-10 |
+| Round of 16 | 2022-04-16/2022-04-17 |
+| Quarterfinals |  2022-04-23/2022-04-24 |
+| Semifinals | 2022-04-30/2022-05-01 |
+| Finals Week 1 | 2022-05-07/2022-05-08 |
+| Finals Week 2/Grand Finals | 2022-05-14/2022-05-15 |
+
```

Comment:
is it the case that the grand finals week overlaps with the finals before? if not then:

```suggestion
| Finals | 2022-05-07/2022-05-08 |
| Grand Finals | 2022-05-14/2022-05-15 |
```

otherwise

```suggestion
| Finals week 1 | 2022-05-07/2022-05-08 |
| Finals week 2 / Grand Finals | 2022-05-14/2022-05-15 |
```